pages/_app.tsx

```diff
@@ -56,6 +57,9 @@ const App = ({ Component, pageProps }: AppProps) => (
   <MsalProvider instance={msalInstance}>
     <IntlProvider locale="en" defaultLocale="en" messages={englishTexts}>
       <Provider store={store}>
+        <Head>
+          <meta name="viewport" content="width=device-width, initial-scale=1" />
```

Comment:
doesn't nextjs add this out of the box?

Reply:
They didn't use to at least, and things suddenly started to look better when I added this 😅